pom.xml

```diff
@@ -290,7 +290,7 @@
             <dependency>
                 <groupId>org.ow2.asm</groupId>
                 <artifactId>asm-bom</artifactId>
-                <version>9.7</version>
+                <version>${dep.asm.version}</version>
```

Comment:
Is this related or does it belong in a separate commit?

Reply:
Doesn't belong here. Fixing